File: exs/mundo_1/c#/006.cs
// crie um algoritmo que leia um número e mostre o seu dobro, o seu triplo e sua raiz quadrada

using System;

// io minusculo para facilitar a leitura, não recomendado
// nomes de classes devem iniciar com a primeira letra maiuscula 
static class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
    public static void Main() {
      double num;
      double dobro;
      double triplo;
      double raizQuadrada;

      num = double.Parse(io.input("Insira um número: "));
      dobro = (num*2);
      triplo = (num*3);
      raizQuadrada = Math.Sqrt(num);

      io.print($"O número digitado foi {num}");
      io.print($"{num} vezes 2 vale {dobro}");
      io.print($"Enquanto seu triplo vale {triplo}");
      io.print($"Já a raiz quadrada é {raizQuadrada}");
    }
}

File: exs/mundo_1/rust/006.rs
/*
Desafio 006

Problema: Crie um algoritmo que leia um número e mostre o seu drobro, triplo e raiz
          quadrada.
*/

fn main() {
    let mut number = String::new();

    println!("Insira um número: ");
    std::io::stdin().read_line(&mut number).unwrap();

    let number: i32 = number.trim().parse().unwrap();

    println!("Dobro: {}", number * 2);
    println!("Triplo: {}", number * 3);
    println!("Raiz quadrada: {}", (number as f64).sqrt());
}
